### PR TITLE
EASY-1975: Pas easy-deposit-ui aan zodat het easy-licenses gebruikt.

### PR DESCRIPTION
**Pull request by rvanheest**
Fixes EASY-1975

#### When applied it will
* Fetch licenses from easy-licenses articraft

#### TODO before merge
* [x] Release `easy-licenses`
* [ ] use a non-SNAPSHOT version of `easy-licenses` in `pom.xml`

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-licenses                      | [PR#1](https://github.com/DANS-KNAW/easy-licenses/pull/1)

### CODE REVIEW

**Comment by rvanheest** [comment]
@janvanmansum @vesaakerman for your consideration: would it make sense to make an NPM package of `easy-licenses` as well and host that on Nexus? Then we can include it as a dependency in `package.json` instead, which means we do not necessarily need Maven to build `easy-deposit-ui`. Of course we keep using it for our convenience, but it feels much more natural to have this in NPM-land.

File: pom.xml [hunk 1]
```diff
@@ -175,7 +175,33 @@
                     </mapping>
                 </configuration>
             </plugin>
-       </plugins>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (janvanmansum) [inline pom.xml]:
```suggestion
                ```

Version is inherited.

File: pom.xml [hunk 2]
```diff
@@ -175,7 +175,33 @@
...
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.8</version>
```
Comment (janvanmansum) [inline pom.xml]:
```suggestion
```
Version is inherited.

File: pom.xml [hunk 3]
```diff
@@ -175,7 +175,33 @@
...
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>nl.knaw.dans.easy</groupId>
+                                    <artifactId>easy-licenses</artifactId>
+                                    <version>1.0.0-SNAPSHOT</version>
+                                    <type>tar.gz</type>
```
Comment (janvanmansum) [inline pom.xml]:
```suggestion
```

easy-licenses has been refactored to a jar.

File: pom.xml [hunk 4]
```diff
@@ -175,7 +175,33 @@
...
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>nl.knaw.dans.easy</groupId>
+                                    <artifactId>easy-licenses</artifactId>
+                                    <version>1.0.0-SNAPSHOT</version>
```
Comment (janvanmansum) [inline pom.xml]: Extract version into property `easy-licenses.version`. (Please, use this convention and *not* ~`easy.licenses.version`~.)

**Pull request merged**